exodus_lambda/functions/origin_response/origin_response.py

```diff
@@ -24,6 +24,12 @@ def handler(self, event, context):
         request = event["Records"][0]["cf"]["request"]
         response = event["Records"][0]["cf"]["response"]
 
+        if "headers" in request and "want-digest" in request["headers"]:
+            sha256_object_key = request["uri"].replace("/", "", 1)
+            response["headers"]["digest"] = [
+                {"key": "Digest", "value": f"id-sha-256={sha256_object_key}"}
```

Comment:
This still needs the base64 encoding previously discussed.

Reply:
I thought id-sha-256 was digest of the content without encoding

Reply:
It's a different encoding being discussed here.

Content-Encoding header can be used in HTTP to encode responses in various ways, e.g. "Content-Encoding: gzip" to indicate that the server has compressed the content and expects the client to decompress.

In that scenario, my understanding is that giving a digest of sha-256 means "This is the SHA256 of the encoded (compressed) data", while giving a digest of id-sha-256 means "This is the SHA256 of the original/decoded data".  The digest header itself still uses base64 encoding, the difference is what data has been checksummed.

Reply:
Okay, I think I understand.